src/main/java/seedu/address/logic/commands/SyncCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import seedu.address.model.Model;
+
+/**
+ * Syncs meetings to the local time on machine.
+ * Deletes meetings with times that are before the present time.
+ */
+public class SyncCommand extends Command {
```

Comment:
I think it should extend UndoableCommand, will be good to undo
Can implement in bugfix phase if needed

Reply:
I agree it would be good 💯 
But implementing it could be much more difficult than for other commands.
I suppose we can keep a copy of the uniquelist, tho that would probably break SLAP quite a bit